docs/architecture.md

```
@@ -0,0 +1,73 @@
+│   ├── layout.tsx  # ルートレイアウト
+│   └── page.tsx    # メインページ
+├── public/         # 静的ファイル
+├── postcss.config.mjs
+├── next.config.ts
+└── package.json
+```
+
+- `line-sample/` は Next.js のサンプルコードです。開発の参考用で、本番では使用しません。
+- `task_memory/` にはタスクログが時系列で保存されています。
+
+## 技術スタック
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)